packages/predeployed-connector/src/react/index.ts

```
@@ -0,0 +1,31 @@
+ * @param options - The options for the predeployed accounts.
+ * @returns The predeployed accounts.
+ */
+export function usePredeployedAccounts(
+    options: PredeployedAccountsConnectorOptions
+) {
+    const [retries, setRetries] = useState<number>(0);
+    const [connectors, setConnectors] = useState<
+        PredeployedAccountsConnector[]
+    >([]);
+
+    useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: retries

This dependency is not specified in the hook dependency list.

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)